src/ott/tools/plot.py

```diff
@@ -119,14 +123,34 @@ def __call__(self, ot: Transport) -> List[plt.Artist]:
     lines = self._mapping(x, y, ot.matrix)
     cmap = plt.get_cmap(self._cmap)
     self._lines = []
+
+    # Make the transparency of the lines depend on the strength of the coupling.
+    adjust_transparency = self._adjust_transparency
+    if adjust_transparency:
+      matrix = ot.matrix
+      min_matrix = float(jnp.min(matrix))
+      max_matrix = float(jnp.max(matrix))
+      scaling_factor = float(jnp.max(jnp.array(matrix.shape)))
+
+      if max_matrix == min_matrix:
+        adjust_transparency = False
+
     for start, end, strength in lines:
+
+      if adjust_transparency:
+        normalized_strength = float(strength) / scaling_factor - min_matrix
+        normalized_strength /= 1e-3 + max_matrix - min_matrix
```

Comment:
Why the `1e-3`? Based on the above code, you won't get a division by zero anymore.